cicecore/cicedynB/infrastructure/ice_blocks.F90

```diff
@@ -252,8 +252,8 @@ subroutine create_blocks(nx_global, ny_global, ew_boundary_type, &
             !*** set last physical point if padded domain
 
             else if (j_global(j,n) == ny_global .and. &
-                     j > all_blocks(n)%jlo      .and. &
-                     j < all_blocks(n)%jhi) then
+                     j >= all_blocks(n)%jlo      .and. &
```

Comment:
If I understand correctly, this change allows jlo=jhi, i.e. 1x1 domains, which weren't allowed before?

Reply:
Correct.  This is where the 1x1 blocks failed.  The 1x1 and 2x2 are still producing different results from the other block sizes and decomps, but they are running and part of our test suite now.

Reply:
Just to elaborate a bit further.  Before this fix, jhi was not set properly if the active block size was 1.  It remained the default (ny_block - nghost), see line 194.    This section of code is where the lo/hi indices are set for padded blocks.  It worked fine as long as the block size was at least 2.